config/locales/en.yml

```diff
@@ -500,7 +500,7 @@ en:
         pledge_unfulfilled: Pledge Not Fulfilled
         resolved: Resolved Without %{fund}
   tooltips:
-    budget_bar: The budget bar lists money tentatively set aside for a patient (by entering a value in the %{fund} pledge field) and money sent to a clinic for a patient (by checking the I sent the pledge checkbox) over a given week. It resets on Mondays.
+    budget_bar: The budget bar lists money tentatively set aside for a patient (by entering a value in the %{fund} pledge field) and money sent to a clinic for a patient (by checking the I sent the pledge checkbox) over a given week. It resets on Mondays by default.
```

Comment:
My inclination was that since there's text showing the start/end days of the current week that would be sufficient, and that anything more might be over-engineering, but I'd be happy to add this. Your call.

Reply:
The point about showing the start/end days of the week is a good one. Let's merge this as is and I'll whip up a quick PR simplifying that i18n text to just take out what is (as you point out here) kinda redundant.